### PR TITLE
Loginovllya rebase

### PR DESCRIPTION
**Pull request by Danack**
Rebased version of #2004 with a few small changes.

### CODE REVIEW

File: reference/imagick/imagickkernel/scale.xml [hunk 1]
```diff
@@ -15,22 +15,41 @@
...
+    </varlistentry>
+    <varlistentry>
+     <term><parameter>normalizeFlag</parameter></term>
+     <listitem>
+      <para>
+       Imagick::NORMALIZE_KERNEL_NONE
+       Imagick::NORMALIZE_KERNEL_VALUE
+       Imagick::NORMALIZE_KERNEL_CORRELATE
+       Imagick::NORMALIZE_KERNEL_PERCENT
+      </para>
```
Comment (cmb69) [inline reference/imagick/imagickkernel/scale.xml:48]: Note that the linebreaks are not retained when this is rendered. You either want to comma separate the constants:
```suggestion
      <para>
       Imagick::NORMALIZE_KERNEL_NONE,
       Imagick::NORMALIZE_KERNEL_VALUE,
       Imagick::NORMALIZE_KERNEL_CORRELATE,
       Imagick::NORMALIZE_KERNEL_PERCENT
      </para>
```

Or use a list:
```suggestion
      <para>
       <simplelist>
        <member>Imagick::NORMALIZE_KERNEL_NONE</member>
        <member>Imagick::NORMALIZE_KERNEL_VALUE</member>
        <member>Imagick::NORMALIZE_KERNEL_CORRELATE</member>
        <member>Imagick::NORMALIZE_KERNEL_PERCENT</member>
       </simplelist> 
      </para>
```

**Review by cmb69 — COMMENTED** [review]
Thank you! This looks good to me, besides the issue pointed out below.

**Review by cmb69 — APPROVED** [review]
Thank you!

**Comment by Danack** [comment]
> of course, our whitespace-checker isn't satisfied when _any_ intermediate commit contains whitespace.

Is it doing the check with git log to only look at modified files?

**Comment by cmb69** [comment]
> aka, the way it's working seems kind of dumb, so there is presumably some reason it's doing it that way, rather than just checking all files for trailing whitespace. Unless there isn't.

It would be horrible for translators to get a single commit with some relevant changes and a huge amount of whitespace changes (which may not even be relevant for them). Thus, we only check actual changes for now.

Anyhow, @TimWolla just landed a [change which does no longer checks intermediate commits](https://github.com/php/doc-en/commit/a08aa8dccbdf96322dac89ab805cbde8097992cd). You can check it out by rebasing (and force-pushing).

**Pull request merged**